sympy/geometry/polygon.py

```diff
@@ -438,6 +438,32 @@ def second_moment_of_area(self, point=None):
         return I_xx, I_yy, I_xy
 
 
+    def polar_modulus(self):
+        """Returns the polar modulus of a two-dimensional polygon"""
+        return self.second_moment_of_area()[0] + self.second_moment_of_area()[1]
+
+
+    def section_modulus(self, point=None):
+        """Returns a tuple with the section modulus of a two-dimensional
+        polygon.
```

Comment:
You should add the description of the `point` argument in docstring and also that its default value is the point farthest from the neutral axis.